src/Clide/LegacyDiagnosticsLogProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.VisualStudio.ProjectSystem;
+using Microsoft.VisualStudio.ProjectSystem.Build;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.BuildLogging;
+
+namespace Clide
+{
+    [AppliesTo("Managed + (Android | iOS | XamarinXaml)")]
```

Comment:
AppliesTo won't be honored for csproj projects. An aggregator that matches the metadata against the project capabilities should work.

Reply:
We already have one such aggregator, the xamarin-provided one